src/plugins/intel_cpu/tests/functional/shared_tests_instances/behavior/infer_request/memory_states.cpp

```diff
@@ -1,24 +0,0 @@
-// Copyright (C) 2018-2023 Intel Corporation
-// SPDX-License-Identifier: Apache-2.0
-//
-
-#include <common_test_utils/test_constants.hpp>
-#include "behavior/infer_request/memory_states.hpp"
-#include "functional_test_utils/plugin_cache.hpp"
-#include "ov_models/builders.hpp"
-
-using namespace BehaviorTestsDefinitions;
-
-namespace {
-std::vector<memoryStateParams> memoryStateTestCases = {
-    memoryStateParams(InferRequestVariableStateTest::getNetwork(), {"c_1-3", "r_1-3"}, ov::test::utils::DEVICE_CPU, {}),
```

Comment:
What is an equivalent for the test in API2.0?

Reply:
This PR only for memory_states: https://github.com/openvinotoolkit/openvino/pull/21627